src/utils.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * utils.ts
+ * Copyright: Microsoft 2019
+ */
+
+export type KeyOrKeys = string | number | string[] | number[];
```

Comment:
this is a breaking change - should be
`string | number | (string | number)[]`

Subtle difference but it does make a difference in complex usages